files/common/etc/default/grub.d/override.cfg

```diff
@@ -118,3 +118,10 @@ GRUB_CMDLINE_LINUX_DEFAULT="$GRUB_CMDLINE_LINUX_DEFAULT init_on_alloc=0"
 # Disable the USB subsystem in its entirety for security reasons.
 #
 GRUB_CMDLINE_LINUX_DEFAULT="$GRUB_CMDLINE_LINUX_DEFAULT usbcore.nousb=1"
+
+#
+# Set the NVME I/O timeout, effectively eliminating it, to align with
+# AWS recommendations. See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html#timeout-nvme-ebs-volumes
+# for details.
```

Comment:
(nit) since this will be part of other platforms like ESX, GCP, etc.., do you mind mentioning that it is ok to have that tunable set for those platforms too? (e.g. due to ZFS we'd be hitting timeout X anyways)

Reply:
Indeed, I've tweaked the comment a little to point this out. Thanks!